examples/benchmarking/README.md

```diff
@@ -0,0 +1,10 @@
+# 🤗 Benchmark results
+
+Here, you can find a list of the different benchmark results created by the community.
+
+If you would like to list benchmark results on your favorite models of the [model hub](https://huggingface.co/models) here, please open a Pull Request and add it below.
+
+| Benchmark description | Results | Environment info |      Author      |
```

Comment:
Is there one number we could put in the actual table. Like tokens/second or MB/1000 tokens so that this table is more than just a ton of links?

Reply:
I'd also have a Framework column instead of including in description. And maybe also `Date`

Reply:
I love this idea though!

Reply:
How good is GitHub at diff'ing large CSV files? I was thinking of copying all results into a single, monster (multi-dimensional?) spreadsheet like @LysandreJik's spreadsheet